packages/phrases/src/locales/de/errors.ts

```diff
@@ -45,15 +45,15 @@ const errors = {
     sign_up_method_not_enabled: 'This sign up method is not enabled.', // UNTRANSLATED
     sign_in_method_not_enabled: 'This sign in method is not enabled.', // UNTRANSLATED
     same_password: 'Das neue Passwort muss sich vom alten unterscheiden.',
-    require_password: 'You need to set a password before sign in.', // UNTRANSLATED
+    require_password: 'You need to set a password before signing-in.', // UNTRANSLATED
     password_exists: 'Your password has been set.', // UNTRANSLATED
-    require_username: 'You need to set a username before sign in.', // UNTRANSLATED
-    username_exists: 'Your username has been set.', // UNTRANSLATED
-    require_email: 'You need to set an email before sign in.', // UNTRANSLATED
-    email_exists: 'Your email has been set.', // UNTRANSLATED
-    require_sms: 'You need to set a phone before sign in.', // UNTRANSLATED
-    sms_exists: 'Your phone has been set.', // UNTRANSLATED
-    require_email_or_sms: 'You need to set a phone or email before sign in.', // UNTRANSLATED
+    require_username: 'You need to set a username before signing-in.', // UNTRANSLATED
+    username_exists: 'The username is already in use.', // UNTRANSLATED
+    require_email: 'You need to add an email address before signing-in.', // UNTRANSLATED
+    email_exists: 'The email address has already been linked to other accounts.', // UNTRANSLATED
```

Comment:
"The email address has already been linked to another account"?
Since an email/sms can only be linked to ONE account in Logto.

Reply:
It should've been `has already been linked to one of the other accounts`, but I just wanted to make it short...

Reply:
haha, np. Just to make sure.